active/gridsyntax/GridSyntaxSpec.md

```diff
@@ -0,0 +1,335 @@
+```
+
+### Grid-specific syntax with CreateFromString attribute assigned 
+The code below has the same functionality as the code shown above with the original syntax, but uses the ColumnDefinition and RowDefinition content property assignments to write it in the following way.
+```xml
+<Grid>
+    <Grid.ColumnDefinitions>
+          <ColumnDefinition>1*</ColumnDefinition>
+          <ColumnDefinition>2*</ColumnDefinition>
+          <ColumnDefinition>Auto</ColumnDefinition>
+          <ColumnDefinition>*</ColumnDefinition>
+          <ColumnDefinition>300</ColumnDefinition>
+    </Grid.ColumnDefinitions>
+    <Grid.RowDefinitions>
+          <RowDefinition>1*</RowDefinition>
+          <RowDefinition>Auto</RowDefinition>
+          <RowDefinition>25</RowDefinition>
+          <RowDefinition>14</RowDefinition>
+          <RowDefinition>20</RowDefinition>
+    </Grid.RowDefinitions>
+</Grid>
+```
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+
+# Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+### Syntax Details and Corner Cases
+The new succinct syntax is formed by assigning a read-only collection-typed property to a collection of values. This collection of values is encased between a set of double quotes. String values, as well as any other more complex type that may require commas within it (i.e. Point objects, sets), are encased between a set of single quotes. This ensures that string values may contain commas, and Points, sets, or any other kind of nested collection object may be supported. To use a single quote within a complex value that is encased in single quotes (i.e., to put a single quote or apostrophe inside of a string value), the HTML entity (&apos;) should be used to escape that character.
+
+### Other Use Cases for Succinct Syntax
+The succinct syntax was created with the goal of making Grid more intuitive. However, this new syntax will be implemented as a language feature and work properly for any read-only collection-typed property. The current syntaxes will still be functional for all other scenarios, as it will be with Grid. Examples of some of these scenarios include, but are not limited to:
```

Comment:
The language feature will work for any collection-typed property, not only those that are read-only.